app/src/androidTest/java/com/swent/assos/ApplicationsTest.kt

```diff
@@ -0,0 +1,114 @@
+package com.swent.assos
+
+import androidx.activity.compose.setContent
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.isDisplayed
+import androidx.compose.ui.test.onNodeWithText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.firebase.Firebase
+import com.google.firebase.firestore.firestore
+import com.swent.assos.model.data.Applicant
+import com.swent.assos.model.data.DataCache
+import com.swent.assos.model.data.Event
+import com.swent.assos.model.data.User
+import com.swent.assos.model.serialize
+import com.swent.assos.screens.ApplicationsScreen
+import com.swent.assos.ui.screens.profile.Applications
+import dagger.hilt.android.testing.HiltAndroidTest
+import io.github.kakaocup.compose.node.element.ComposeScreen
+import java.time.LocalDateTime
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@HiltAndroidTest
+class ApplicationsTest : SuperTest() {
+
+  private val assoId = "02s16UZba2Bsx5opTcQb"
+  private val eventId = "eventId"
+  private val user =
+      User(
+          id = "11111",
+          firstName = "Paul",
+          lastName = "Levebre",
+          email = "test@epfl.ch",
+          associations = emptyList(),
+          sciper = "330245",
+          semester = "GM-BA2",
+          following = emptyList(),
+          tickets = emptyList(),
+          appliedAssociation = listOf(assoId),
+          appliedStaffing = listOf(eventId))
+
+  override fun setup() {
+    super.setup()
+    DataCache.currentUser.value = user
+    Firebase.firestore
+        .collection("associations")
+        .document(assoId)
+        .set(
+            mapOf(
+                "acronym" to "Polylan",
+                "fullname" to "Polylan",
+                "url" to "https://polylan.ch",
+                "description" to "Polylan is the biggest lan party in Switzerland",
+                "logo" to "https://polylan.ch/logo.png",
+                "banner" to "https://polylan.ch/banner.png"))
+
```

Comment:
If I am not mistaken, all the association are already on the firestore database. Hence, you do not need to set the content of the association.

Reply:
Thank you, I fixed it.